nvdaHelper/interfaces/nvdaInProcUtils/nvdaInProcUtils.idl

```diff
@@ -68,6 +68,12 @@ interface NvdaInProcUtils {
 
 	error_status_t sysListView32_getColumnContent([in] const unsigned long windowHandle, [in] int item,  [in] int subItem, [out,string] BSTR* text);
 
+	error_status_t sysListView32_getColumnLocation([in] const unsigned long windowHandle, [in] int item,  [in] int subItem, [out] RECT* location);

```

Comment:
You can use the following to add docs to the generated header file, I'd suggest mentioning the name is adjusted when the headers are generated at least.
```suggestion
	cpp_quote("/* Actually maps to nvdaInProcUtils_sysListView32_getColumnLocation")
	cpp_quote("*/")
	error_status_t sysListView32_getColumnLocation([in] const unsigned long windowHandle, [in] int item,  [in] int subItem, [out] RECT* location);

```